src/axios.js

```diff
@@ -2,7 +2,7 @@ const Axios = require("axios");
 const Cookies = require("js-cookie");
 
 const baseDomain =
-  process.env.NEXT_PUBLIC_BACKEND_BASE_DOMAIN || "https://api.omnistrate.cloud";
+  process.env.NEXT_PUBLIC_BACKEND_BASE_DOMAIN || "https://api.omnistrate.dev";
```

Comment:
@mbianchidev we can't commit. This is an external package and ".dev" is an internal endpoint. This change will break production for all of our customers.

Reply:
Shouldn't they set the env var NEXT_PUBLIC_BACKEND_BASE_DOMAIN? 
Anyway, good to know!